dataactcore/migrations/versions/c75d57250419_fileF_add_state_name.py

```diff
@@ -0,0 +1,49 @@
+"""Adding schema v1.2 F columns
```

Comment:
Generally, when adding these messages, we don't want to say "F columns" or something because that's not really indicative of the database. We want to stick to facts about the migration like "Adding state name columns to all fsrs tables" or something.

Reply:
Updated Description and file name